app/models/booking.rb

```diff
@@ -2,7 +2,7 @@ class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :service
 
-  validates :user_id, :service_id, :start_date, :end_date, presence: true
+  validates :user_id, :service_id, :start_date, :end_date, :name, presence: true
```

Comment:
Is this name referring to the service_name or the user's first/last name?

Reply:
yea this needs to be changed and updated, I confused it with another part of the table and thought it can grab the names based on the user_id as below:
![image](https://github.com/HappyDay101/Friends-4-Benefits/assets/132921363/da494ddc-687f-41c4-9cfb-fa3eb4aa8afc)

I will have to add the User`s names to the booking model

Reply:
I think the user's name doesn't need to be added to the booking model because it is already connected through as a foreign key through the "belongs_to" declaration at the top. 

I've been making the connection with "booking.user.first_name" as an example.